libuavcan_drivers/stm32/driver/include/uavcan_stm32/thread.hpp

```diff
@@ -17,6 +17,12 @@
 # include <ctime>
 # include <cstring>
 #elif UAVCAN_STM32_BAREMETAL
+#elif UAVCAN_STM32_FREERTOS
+# include <stm32f4xx.h>
+# include <cmsis_os.h>
+  #ifndef MAX_SEMAPHORE_COUNT
+    #define MAX_SEMAPHORE_COUNT      50
+  #endif
```

Comment:
Please fix formatting above: the octothorpe symbol (`#`) should be placed on the beginning of the line.